api-client/src/models/DateModel.ts

```diff
@@ -99,6 +101,14 @@ class DateModel {
     return `${localizedStartDate} - ${localizedEndDate.toFormat(format)}`
   }
 
+  private rruleToDateTime(date: Date): DateTime {
+    const dateTime = DateTime.fromJSDate(date)
+    // rrule is not correctly handling timezones and always returning local time in the shape of UTC
+    // Therefore we have to manually apply timezone offset, e.g. for daylight savings time
+    // https://github.com/jkbrzt/rrule#important-use-utc-dates
```

Comment:
This link suggest to handle this with luxon, like this:
```
rule.all().map(date =>
DateTime.fromJSDate(date)
  .toUTC()
  .setZone('local', { keepLocalTime: true })
  .toJSDate()
)
```

Why have you decided to handle it like you did?